samples/tab-staggered-permission/csharp/StaggeredPermission/helper/AuthHelper.cs

```diff
@@ -83,8 +83,8 @@ public static IEnumerable<string> GetValidIssuers(IConfiguration configuration)
         public static async Task<string> GetAccessTokenOnBehalfUserAsync(IConfiguration configuration, IHttpClientFactory httpClientFactory, IHttpContextAccessor httpContextAccessor, string idToken)
         {
             var tenantId = configuration["AzureAd:TenantId"];
-            IConfidentialClientApplication app = ConfidentialClientApplicationBuilder.Create(configuration["AzureAd:MicrosoftAppId"])
-                                                .WithClientSecret(configuration["AzureAd:MicrosoftAppPassword"])
+            IConfidentialClientApplication app = ConfidentialClientApplicationBuilder.Create(configuration["AzureAd:ClientId"])
+                                                .WithClientSecret(configuration["AzureAd:AppSecret"])
```

Comment:
check appsettings.json not found  "AppSecret" and "ClientId"

Reply:
fixed